CODEOWNERS

```
@@ -76,8 +76,8 @@ go.mod @fleetdm/go
 #
 # (see website/config/custom.js for DRIs of other paths not listed here)
 ##############################################################################################
-/docs                                           @rachaelshaw
-/docs/Using-Fleet/REST-API.md                   @rachaelshaw # « REST API reference documentation
+/docs                                           @rachaelshaw @eashaw @noahtalerman
+/docs/Using-Fleet/REST-API.md                   @rachaelshaw @lukeheath # « REST API reference documentation
```

This file has the old location of the REST API docs
```suggestion
/docs/REST\ API/rest-api.md                   @rachaelshaw @lukeheath # « REST API reference documentation
```